circus-train-api/pom.xml

```diff
@@ -61,6 +61,7 @@
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-exec</artifactId>
+      <classifier>core</classifier>
```

Comment:
Why do we need to depend on the "core" classifier now?

Reply:
hive-exec (is an uberjar)
hive-exec-core is the actual hive-exec hive code. We should if we can never rely on the non-core jar as it pulls in loads of stuff, guava among others.

Reply:
Agreed, makes sense.